src/formats/gif.zig

```diff
@@ -227,10 +237,85 @@ pub const GIF = struct {
     }
 
     pub fn writeImage(allocator: std.mem.Allocator, write_stream: *io.WriteStream, image: Image, encoder_options: Image.EncoderOptions) Image.WriteError!void {
-        _ = allocator;
-        _ = write_stream;
-        _ = image;
-        _ = encoder_options;
+        const writer = write_stream.writer();
```

Comment:
There is no way to write GIF without using `Image`, you should have a `write` function that accepts a slice of `AnimationFrame`  that this function would use.